core/src/main/java/io/confluent/rest/Application.java

```diff
@@ -370,11 +378,9 @@ public Handler configureHandler() {
 
     RequestLogHandler requestLogHandler = new RequestLogHandler();
     requestLogHandler.setRequestLog(requestLog);
+    context.insertHandler(requestLogHandler);
```

Comment:
this is the main logic of this PR, simply inserting the requestloghandler into context handler.